prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-basis: calc(100% / 3);
```

Comment:
Careful `flex-basis` goes on flex items (children of elements with `display: flex`). You can add another rule set with a selector for `.card-container` and apply the `flex-basis` there.